README

```diff
@@ -1,29 +1,34 @@
-
-SQUID Web Proxy Cache                         http://www.squid-cache.org/
--------------------------------------------------------------------------
+SQUID Web Proxy Cache                        http://www.squid-cache.org/
```

Comment:
seems reasonable. and thanks for the heads up on 72, will adjust my tooling a bit.